README.md

```diff
@@ -38,6 +38,12 @@ conda install -c conda-forge r-grf
 
 Note that to install from source, a compiler that implements C++11 is required (clang 3.3 or higher, or g++ 4.8 or higher). If installing on Windows, the RTools toolchain is also required.
 
+The current development version can be installed from source using devtools.
+```R
+library(devtools)
+install_github(‘grf-labs/grf’, subdir=‘r-package/grf’)
```

Comment:
I think you need to use double quotes instead, if I copy and paste this into R I get an error 

```
> install_github(‘grf-labs/grf’, subdir=‘r-package/grf’)
Error: unexpected input in "install_github(�"
```

Reply:
Yeah, not sure what I was thinking. Thanks.

Reply:
Another tiny nitpick is we can get rid of one line by just doing `devtools::install_github`, but do not have any strong opinion, you decide! (I hit approved)

Reply:
+1 to @erikcs's nitpick :)

Reply:
Another nitpick is space between `=`